show/main.py

```diff
@@ -1005,12 +1006,20 @@ def platform():
 
 # 'summary' subcommand ("show platform summary")
 @platform.command()
-def summary():
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+@click.option('--json', is_flag=True, help="JSON output")
+def summary(verbose, json):
     """Show hardware platform information"""
+
     hw_info_dict = get_hw_info_dict()
-    click.echo("Platform: {}".format(hw_info_dict['platform']))
-    click.echo("HwSKU: {}".format(hw_info_dict['hwsku']))
-    click.echo("ASIC: {}".format(hw_info_dict['asic_type']))
+    if json:
+        click.echo(hw_info_dict)
+    else:
+        click.echo("Platform: {}".format(hw_info_dict['platform']))
+        click.echo("HwSKU: {}".format(hw_info_dict['hwsku']))
+        click.echo("ASIC: {}".format(hw_info_dict['asic_type']))
+        if verbose:
+            click.echo("ASIC Count: {}".format(hw_info_dict['asic_count']))
```

Comment:
I think it may be better to always display this line without the need for a `--verbose` flag. Just need to check the rest of the SONiC codebase to see if this output is parsed anywhere and make sure it is smart enough to handle an additional line.

Does anyone disagree?

Reply:
At this point it may seem redundant for most cases, but I prefer that too - show by default.